core/src/test/java/com/hyperwallet/android/model/HyperwalletBankAccountQueryParamTest.java

```diff
@@ -173,5 +170,11 @@ public void testBuilder_verifyValues() {
         assertThat(queryParam.getType(), is(BANK_ACCOUNT));
         assertThat(queryParam.getCreatedAfter().getTime(), is(dateAfter.getTimeInMillis()));
         assertThat(queryParam.getCreatedBefore().getTime(), is(dateBefore.getTimeInMillis()));
+
+        queryParam = new HyperwalletBankAccountQueryParam.Builder()
```

Comment:
we don't need this part of the code. There's an assertion checking the type in line 170. Also in this test there's a calendar variable called dateOn that is not being used and can be removed

Reply:
@fmattos-hw We need this part of the test to increase test coverage for HyperwalletBankAccountQueryParam.Builder because we have such case:
if (mType == null) {
  type(BANK_ACCOUNT);
}
I faced with jacoco error: Rule violated for class com.hyperwallet.android.model.transfermethod.HyperwalletBankAccountQueryParam.Builder: branches covered ratio is 0.50, but expected minimum is 0.65

Reply:
I was able to fix it by invoking the type method in the builder 
```suggestion
HyperwalletBankAccountQueryParam queryParam = new HyperwalletBankAccountQueryParam.Builder()
                .offset(100)
                .limit(20)
                .type(BANK_ACCOUNT)
```